index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/style.css">
+    <script src="https://use.fontawesome.com/bba5c6ae87.js"></script>
+    <script src="https://use.fontawesome.com/83ac61cd0f.js"></script>
+    <title>My Portfolio</title>
+</head>
+<body>
+<header class="wrapper">
+    <nav class="flex-nav">
+        <ul>
+            <li ><a class="logo" href="#">Chidiebere</a> </li>
+            <li class="menu-icon toggle-icon"><i class="fa fa-bars" aria-hidden="true"></i></li>
+        </ul>
+    </nav>
+</header>
+
+
+<section>
+    <div class="headline">
+
+        <h1 class="intro name">Hey There. I'm Chidiebere Ojingwa</h1>
+        <p class="intro profession">I'am a Software Developer</p>
+        <p class="intro task"> I can help you build a product, feature or <br> website
+        look through some of my work <br>and experience! If you like what you see <br> and
+        have a project you need, <br> don't hesitate to contact me</p>
+
+        <div class="social">
+            <a href="#" class="fa fa-github"></a>
+            <a href="#" class="fa fa-linkedin"></a>
+            <a href="#" class="fa fa-angellist"></a>
+            <a href="#" class="fa fa-twitter"></a>
+            <a href="#" class="fa fa-medium"></a>
+        </div>
+    </div>
+
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and` li` per icon; each icon has a link (`a tag`). Please, let's update that. :pray: